ethdk/src/Bls/BlsNetworks.ts

```diff
@@ -0,0 +1,20 @@
+import { type BlsNetwork } from '../interfaces/Network'
```

Comment:
I thought it would be good to have a networks file for each account type we add.  That way we can dictate which networks a dev can create an account on